proxy/src/serverless/sql_over_http.rs

```diff
@@ -179,10 +182,23 @@ fn get_conn_info(
     }
     ctx.set_user(username.clone());
 
-    let password = connection_url
-        .password()
-        .ok_or(ConnInfoError::MissingPassword)?;
-    let password = urlencoding::decode_binary(password.as_bytes());
+    let auth = if let Some(auth) = headers.get(&AUTHORIZATION) {
+        let auth = auth
+            .to_str()
+            .map_err(|_| ConnInfoError::InvalidHeader(&AUTHORIZATION))?;
+        AuthData::Jwt(
```

Comment:
Is there a test that checks the requests with just "Bearer " and nothing following it won't make it far? I guess at least the deser will fail.

Reply:
I guess I could add a fast path for both password and jwt being empty, but it feels rather niche since both flows should eventually disregard them. I need to add some integration tests for the local-proxy and I can include a test for that there.